src/cocotb_tools/config.py

```diff
@@ -47,21 +47,27 @@
 
 import find_libpython
 
-import cocotb
+import cocotb_tools
 
-__all__ = ["share_dir", "makefiles_dir", "libs_dir"]
+base_tools_dir = Path(cocotb_tools.__file__).parent.resolve()
+base_cocotb_dir = base_tools_dir.parent.joinpath("cocotb").resolve()
```

Comment:
Add check to see if this exists and if not then `import cocotb` and use that instead.